com.unity.ml-agents/CONTRIBUTING.md

```diff
@@ -48,18 +48,18 @@ nature of the environment and task.
 
 ## Continuous Integration (CI)
 
-We run CircleCI on all PRs; all tests must be passing before the PR is merged.
+We run continuous integration on all PRs; all tests must be passing before the PR is merged.
 
 Several static checks are run on the codebase using the
 [pre-commit framework](https://pre-commit.com/) during CI. To execute the same
-checks locally, install `pre-commit` and run `pre-commit run --all-files`. Some
-hooks (for example, `black`) will output the corrected version of the code;
+checks locally, install `pre-commit` via `pip` and run `pre-commit run --all-files`.
```

Comment:
maybe add the whole pip install command explicitly to save someone a syntax lookup 
`pip install pre-commit`

Reply:
Updated.